clang/test/CodeGenCXX/vtable-assume-load-address-space.cpp

```diff
@@ -23,8 +26,8 @@ struct B : A {
 void g(A *a) { a->foo(); }
 
 // CHECK1-LABEL: define{{.*}} void @_ZN5test14fooAEv()
-// CHECK1: call void @_ZN5test11AC1Ev(ptr
-// CHECK1: %[[VTABLE:.*]] = load ptr addrspace(1), ptr %{{.*}}
+// CHECK1: call{{.*}} void @_ZN5test11AC1Ev(ptr {{((addrspace(4)){0,1})}}
```

Comment:
I don’t love how this isn’t checking the address space is correct for each, only that it’s one of the two valid ones

Reply:
Hmm, I can rework the regexp (it would still be icky), but it's not going to help with the issue I don't think. I would prefer to not spam another prefix, which is the only other way I can see for dealing with this without duplicating the test, but if the lack of love is extreme I can probably go that way.

Reply:
I'd say this is probably fine for now, but these tests should be deduplicated and cleaned up in a follow-up.

Reply:
Right, so what I'll do is: merge this as is, and then revisit the test and rework it along the lines you and @jrtc27 are suggesting, since it looks as if I'll need to mess with it again for another piece of work.